### ISSUE TITLE
MD075 erronoeously triggers on math absolute value bars

### ISSUE THREAD

**Issue by vEnhance** [rvben/rumdl#557]
### rumdl version

0.1.65

### Installation method

Pre-built binary

### Bug description

Absolute value bars in math mode also trigger MD075 erroneously.
```markdown
# Math

Roughly (for privacy reasons, this isn't exactly what the student said),
the student talked about having done small cases on the size $|S|$,
and figuring out that $|S|$ was even, but then running out of ideas.
```

### Steps to reproduce

```
$ rumdl --version
rumdl 0.1.65
$ ls
math.md
$ rumdl check --no-config math.md
math.md:4:1: [MD075] Pipe-formatted rows without a table header/delimiter row

Issues: Found 1 issues in 1 file (23ms)
```

### Expected behavior

No error should be reported.

### Actual behavior

MD075 is reported

### Configuration

```toml

```

### Operating system

Linux

**Comment by rvben** [comment]
Thanks for the report, @vEnhance! Fixed in v0.1.66 — pipes inside `$...$` inline math are no longer treated as table separators.
